src/data/fintype/basic.lean

```diff
@@ -318,6 +318,10 @@ lemma finset.card_compl [decidable_eq α] [fintype α] (s : finset α) :
   sᶜ.card = fintype.card α - s.card :=
 finset.card_univ_diff s
 
+lemma finset.compl_to_finset [decidable_eq α] [fintype α] (s : set α) :
```

Comment:
I think this should maybe be called `set.to_finset_compl`, and live in `data/set/finite` next to `set.to_finset_inter`?

Reply:
That might make more sense actually, yeah. I'll move it

Reply:
Ah, I did it that way because in #5698 I'm working with subtypes. I'll revisit this later to sort out whether I actually need the original version.

Reply:
Actually scratch that - I'll let this PR stay up because I do think this is worth adding to mathlib regardless of whether I use it, I'll just have to figure out those details later